ethereum/spec/src/main/java/tech/pegasys/teku/spec/genesis/GenesisGenerator.java

```diff
@@ -98,6 +98,7 @@ public void updateCandidateState(
                         UInt64.ZERO,
                         UInt64.ZERO,
                         UInt64.ZERO,
+                        Bytes.wrap(new byte[SpecConfig.MAX_EXTRA_DATA_BYTES]),
```

Comment:
```suggestion
                        Bytes.Bytes.EMPTY,
```
The default value for an `SszList` would be empty list